raiden/settings.py

```diff
@@ -25,3 +25,6 @@
 DEFAULT_NAT_KEEPALIVE_RETRIES = 5
 DEFAULT_NAT_KEEPALIVE_TIMEOUT = 30
 DEFAULT_NAT_INVITATION_TIMEOUT = 180
+
+DEFAULT_INITIAL_HEALTHCHECK_TIMEOUT = 5
```

Comment:
Please, change `--max-unresponsive-time` at the raiden ui to 25 to get the same effect.

Reply:
Humm, actually `--send-ping-time` needs fixing, it's not using the default value

Reply:
With --max-unresponive time to `25` it's still quite a long waiting time ([60, 60, 60, 60, 60])
and I argue that the initial ping should be a lot more strict than the later ones.